crates/re_viewer/src/ui/selection_history_ui.rs

```diff
@@ -157,12 +157,18 @@ fn item_kind_ui(ui: &mut egui::Ui, sel: &Item) {
     ui.weak(RichText::new(format!("({})", sel.kind())));
 }
 
-fn item_collection_to_string(blueprint: &ViewportBlueprint, items: &ItemCollection) -> String {
-    assert!(!items.is_empty()); // history never contains empty selections.
-    if items.len() == 1 {
-        item_to_string(blueprint, items.iter().next().unwrap())
-    } else if let Some(kind) = items.are_all_same_kind() {
-        format!("{}x {}s", items.len(), kind)
+fn selection_to_string(blueprint: &ViewportBlueprint, selection: &Selection) -> String {
+    debug_assert!(!selection.is_empty()); // history never contains empty selections.
```

Comment:
Here and everywhere else: any comment on an assert statement should really just be part of the assertion imo
```suggestion
    debug_assert!(!selection.is_empty(), "history never contains empty selections");